src/app/data-analytics-dashboard/oncology/oncology-reports/oncology-indicators-patient-list/oncology-indicators-patient-list.component.ts

```diff
@@ -50,14 +50,14 @@ export class OncologySummaryIndicatorsPatientListComponent implements OnInit {
 
   public ngOnInit() {
     this.route.queryParams.subscribe((params: any) => {
-        if (params) {
-          this.getPatientList(params);
-          this.title = this.translateIndicator(params.indicators);
-          this.params = params;
-        }
-      }, (error) => {
-        console.error('Error', error);
-      });
+      if (params) {
+        this.getPatientList(params);
+        this.title = this.translateIndicator(params.indicators);
+        this.params = params;
+      }
+    }, (error) => {
+      console.error('Error', error);
+    });
```

Comment:
If you find time. Can you come up with the code formatting standard for this project?😊

Reply:
Yeah, looks like my linter had fun here. I think it's time we had a conversation about linting ng2-amrs.

Reply:
Are you ready to take up the task?

Reply:
Someone's got to do it. I'll ask Nicky about his thoughts on the matter.